compiler/one-cmds/onecc.template.cfg

```diff
@@ -1,29 +1,93 @@
+; To activate each step(or task),
+; change to True for each [onecc] items and fill sub-items in each corresponding section
 [onecc]
-one-import-tf=True
+; neural network model to circle
+one-import-tf=False
 one-import-tflite=False
 one-import-bcq=False
 one-import-onnx=False
-one-optimize=True
+; circle to circle with optimization
+one-optimize=False
+; circle to circle with quantization
 one-quantize=False
+; partition circle
 one-partition=False
-one-pack=True
+; package circle and metadata into nnpackage
+one-pack=False
+; codegen from circle to backend's type
 one-codegen=False
+; profile
 one-profile=False
 
 [one-import-tf]
-input_path=/path/to/inception_v3.pb
-output_path=inception_v3.circle
-input_arrays=input
-input_shapes=1,299,299,3
-output_arrays=InceptionV3/Predictions/Reshape_1
-converter_version=v1
-model_format=graph_def
+; mandatory
+input_path=             ; pb file
+output_path=            ; circle file
+; optional but mandatory for v1
+input_arrays=           ; input tensor names
+input_shapes=           ; eg) 1,299,299,3
+output_arrays=          ; output tensor names
+; optional
+converter_version=      ; converter version: v1 or v2
+model_format=graph_def  ; graph_def, saved_model or keras_model
+
+[one-import-tflite]
+; mandatory
+input_path=        ; tflite file
+output_path=       ; circle file
+
+[one-import-bcq]
+; mandatory
+input_path=        ; bcq file
+output_path=       ; circle file
+input_arrays=      ; input tensor names
+input_shapes=      ; eg) 1,299,299,3
+output_arrays=     ; output tensor names
```

Comment:
Maybe mandatory for v1? like `[one-import-tf]` ?

Reply:
✔️